ui/src/cohort.ts

```diff
@@ -39,6 +39,7 @@ export enum GroupKind {
 
 export interface Group {
   id: string;
+  name: string;
```

Comment:
Make this optional so it doesn't have to be filled since it always starts empty.

Reply:
good catch